assets/js/main.js

```diff
@@ -141,11 +136,10 @@ $(document).ready((function ($) {
             host = 'http://127.0.0.1:8000/',
             path = 'api/signin/',
             url = host + path;
-
         $.ajax({
             type: 'POST',
             url: url,
-            data: JSON.stringify(data),
+            data: data,
```

Comment:
Why did you remove `JSON.stringify` ?

<img width="796" alt="Screenshot 2021-12-31 at 00 10 02" src="https://user-images.githubusercontent.com/12160621/147791124-330a251b-3b6f-4404-bc83-9f3adb64b105.png">

Now I can't sign in. :(

Reply:
Done